ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/PerasCertDB/Impl.hs

```diff
@@ -144,7 +144,7 @@ implAddCert ::
   , StandardHash blk
   ) =>
   PerasCertDbEnv m blk ->
-  PerasCert blk ->
+  ValidatedPerasCert blk ->
```

Comment:
Currently, this function uses `boostPerCert`; rather, it should now use `validatedPerasCertBoost` of the argument here.